crates/ruff_server/src/server/api/requests/execute_command.rs

```diff
@@ -37,11 +37,11 @@ impl super::SyncRequestHandler for ExecuteCommand {
             let output = debug_information(session);
             notifier
                 .notify::<types::notification::LogMessage>(types::LogMessageParams {
-                    message: output,
+                    message: output.clone(),
                     typ: types::MessageType::INFO,
                 })
                 .with_failure_code(ErrorCode::InternalError)?;
```

Comment:
I prefer to remove the logging part and I don't think it should break anything.

Reply:
I'm in favor of removing it, but wouldn't it break old VS code clients that don't have the required code to open the document on the client side?

Reply:
Hmm yeah. So, that would happen if the user is using a non-latest extension version with the latest ruff in which case I think we should recommend them to upgrade the extension version as it always include improvements and bug fixes. I'm fine keeping this here as well for the time being and we can remove it with 0.10.

Reply:
Can we include a `TODO` or similar that it should be removed? Or do we want to keep it, considering that other clients can't easily call commands?

Reply:
I think we should remove it in a later version, I'll create an issue for 0.10.